.changeset/strange-plants-pull.md

```diff
@@ -0,0 +1,5 @@
+---
+'contexture-elasticsearch': minor
+---
+
+Pivot: configuring `groupCounts` and `skip` per group/value level
```

Comment:
Can you add a better description? What does `groupCounts` do? What does `skip` do?

Reply:
Agreed - this should also be in the readme.